packages/flutter/lib/src/cupertino/text_form_field.dart

```diff
@@ -0,0 +1,393 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:flutter/rendering.dart';
+import 'package:flutter/services.dart';
+import 'package:flutter/widgets.dart';
+
+import 'colors.dart';
+import 'split_form_row.dart';
+import 'text_field.dart';
+import 'theme.dart';
+
+export 'text_form_field.dart';
+
+/// Creates a [FormField] that contains a [CupertinoTextField].
+///
+/// This is a convenience widget that wraps a [CupertinoTextField] widget in a
+/// [FormField].
+///
+/// A [Form] ancestor is not required. The [Form] simply makes it easier to
+/// save, reset, or validate multiple fields at once. To use without a [Form],
+/// pass a [GlobalKey] to the constructor and use [GlobalKey.currentState] to
+/// save or reset the form field.
+///
+/// When a [controller] is specified, its [TextEditingController.text]
+/// defines the [initialValue]. If this [FormField] is part of a scrolling
+/// container that lazily constructs its children, like a [ListView] or a
+/// [CustomScrollView], then a [controller] should be specified.
+/// The controller's lifetime should be managed by a stateful widget ancestor
+/// of the scrolling container.
+///
+/// A [leadingText] is required. This string is displayed in a [Text] widget
+/// leading to the side of the [CupertinoTextFormFieldRow]. It can be set to
+/// an empty string, although it is encouraged to set the [leadingText] in
+/// order to follow standard iOS practices.
+///
+/// If a [controller] is not specified, [initialValue] can be used to give
+/// the automatically generated controller an initial value.
+///
+/// Remember to call [TextEditingController.dispose] of the [TextEditingController]
```

Comment:
We have this text in other files too but it's slightly awkward. Better to state `Remember to call [TextEditingController.dispose] of the [controller], if one is specified, when it is no longer needed`. Otherwise it's confusing as to what [TextEditingController] we're referring to if the user never used one anywhere.